src/Concerns/ManagesInvoices.php

```diff
@@ -160,11 +160,18 @@ public function createInvoice(array $options = [])
     {
         $this->assertCustomerExists();
 
+        $stripeCustomer = $this->asStripeCustomer();
+
         $parameters = array_merge([
             'automatic_tax' => $this->automaticTaxPayload(),
             'customer' => $this->stripe_id,
+            'currency' => $stripeCustomer->currency ?? config('cashier.currency'),
```

Comment:
There cannot be multiple currencies anymore on invoices so we'll use the one from the customer or fallback to the Cashier default.